v0/pipeline.json

```diff
@@ -14536,8 +14536,7 @@
                   "items" : {
                     "type" : "string"
                   },
-                  "maxItems" : 2147483647,
-                  "minItems" : 1
```

Comment:
why are we removing minItems ?

Reply:
min items indicate that if the field is mentioned, the array cannot be empty. It should atleast have 1 element. 

Now we want to support the following use-case, 
approvers: 
  userGroups: []
  serviceAccounts:  
    - abc

This was not supported earlier because of minItems:1. We have removed that and added extra validations below disallowing both to be empty simultaneously.